### PR TITLE
Mango_lassi recipe

### PR DESCRIPTION
**Pull request by saikoushalg**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, can you rename the files to the following so they don't conflict with https://opendrinks.io/recipe/mango-lassi:
```
creamy-mango-lassi.jpg
creamy-mango-lassi.json
```

**Comment by saikoushalg** [comment]
> 
> 
> Hi, can you rename the files to the following so they don't conflict with https://opendrinks.io/recipe/mango-lassi:
> 
> ```
> creamy-mango-lassi.jpg
> creamy-mango-lassi.json
> ```

i have renamed the files please check.

File: src/recipes/creamy-mango-lassi.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "directions": [
+        " Take a large bowl, add 250ml  fresh mango pulp.",
+        "To the same bowl, now add yogurt.",
+        "Now add cold milk.",
+        "Also add the sugar and the cardamom powder.",
+        "Blend everything until well combined.",
+        "Pour the mango lassi into serving glasses and chill before serving",
+        " garnish with crushed almonds,before serving! " 
+    ],
+    "image": "Mango_lassi.jpg",
```
Comment (alfg) [inline src/recipes/creamy-mango-lassi.json:49]:
```suggestion
    "image": "creamy-mango-lassi.jpg",
```

**Comment by alfg** [comment]
Thank you!

**Pull request merged**